tests/test_base.py

```diff
@@ -0,0 +1,52 @@
+import json
+import os
+import datetime
+from typing import List
+from config import path_config
+from models.input import Input
+from models.output import Output
+from transform.student.student_transformer import StudentTransformer
+from send_generated_input_script.rabbit_mq_publisher import RabbitMqPublisher
+from tests.test_model.test_input import TestInput
+
+
+class TestBase:
+
+    STUDENTS_DIR_PATH = path_config.STUDENTS_DIR_PATH
```

Comment:
why absolute and not relative ?

Reply:
i tried to do it relative but ran into problems
i know that it isnt best practice, ill read about it